apps/weshipit/pages/index.tsx

```diff
@@ -25,11 +24,14 @@ export default function IndexPage({ clients }: clientsPageProps) {
       <Hero />
       <section className="m-auto max-w-7xl">
         <div className="my-48 text-center">
-          <Text variant="s2" style="uppercase">
+          <Text variant="h3" style="py-4">
             Some of the wonderful clients who have trusted us
           </Text>
           <ClientsListHomepage clients={clients} />
-          <Link href="/clients">View our client list</Link>
+
+          <Link href="/clients" style="m-auto w-52  ">
+            View our client list
```

Comment:
to style you can use 

```
 <a className="text-sm hover:text-gray-600">Test Link styling</a>
```

https://stackoverflow.com/questions/69889572/tailwindcss-isnt-applying-to-nextjs-link-component